modules/versioning.nix

```
@@ -169,6 +169,23 @@ let
             This includes the final database size (0.4) plus some extra storage (0.2).
       '';
     }
+    {
+      version = "0.0.57";
+      condition = config.services.liquidd.enable && config.services.liquidd.prune == 1000;
+      message = ''
+         With this version, the `secure-node.nix` preset does _not_ set
+         `liquidd.prune=1000` anymore. If you are using the preset (i.e. you
+         import `secure-node.nix` in your configuration.nix), you must take one
+         of the following actions:
+
+           - If you want to keep the same behavior as before, set manually set
+             `services.liquidd.prune = 1000;` in your configuration.nix.
+           - Otherwise, if you want to turn off pruning, you must instruct liquidd
+             to reindex by setting `services.liquidd.extraConfig = "-reindex";`.
```

```suggestion
             to reindex by setting `services.liquidd.extraConfig = "reindex=1";`.
```
`-reindex` only works as a cmdline arg.